examples/send-transfers.rs

```diff
@@ -0,0 +1,63 @@
+//! Send a transfer to an address.
+//!
+//! Run with:
+//!
+//! ```
+//! cargo run --example send-transfers
+//! ```
+use anyhow::Result;
+use iota::{
+    bundle::{Address, TransactionField},
+    client::Transfer,
+    crypto::Kerl,
+    signing::{IotaSeed, Seed},
+    ternary::{T1B1Buf, TryteBuf},
+};
+
+#[smol_potat::main]
+async fn main() -> Result<()> {
+    // Prepare a vector of transfers
+    let mut transfers = Vec::new();
+
+    // Push the transfer to vector.
+    transfers.push(Transfer {
+        // Address is 81 trytes.
+        address: Address::from_inner_unchecked(
+            TryteBuf::try_from_str(
+                "RVORZ9SIIP9RCYMREUIXXVPQIPHVCNPQ9HZWYKFWYWZRE9JQKG9REPKIASHUUECPSQO9JT9XNMVKWYGVA",
+            )
+            .unwrap()
+            .as_trits()
+            .encode(),
+        ),
+        // We are using a zero balance seed so we make a zero value transfer here
+        value: 0,
+        message: "".to_string(),
+    });
+
+    // Create a client instance
+    let res = iota::Client::new("https://nodes.comnet.thetangle.org")
+        // Call send_transfers api
+        .send_transfers()
+        // Input your seed
+        .seed(
+            &IotaSeed::<Kerl>::from_buf(
+                TryteBuf::try_from_str("RVORZ9SIIP9RCYMREUIXXVPQIPHVCNPQ9HZWYKFWYWZRE9JQKG9REPKIASHUUECPSQO9JT9XNMVKWYGVA")
```

Comment:
the seed is the same as address might confuse users, adds a comment or use a dummy seed?

Reply:
Let me add more comment about this then.